frontend/src/components/layout/navigation/whatsnew/WhatsNewContent.tsx

```diff
@@ -1,10 +1,25 @@
+import { OutboundLink } from "react-ga";
+import {
+  getPremiumSubscribeLink,
+  isPremiumAvailableInCountry,
+} from "../../../../functions/getPlan";
+import { useProfiles } from "../../../../hooks/api/profile";
+import { useRuntimeData } from "../../../../hooks/api/runtimeData";
+import { useGaViewPing } from "../../../../hooks/gaViewPing";
 import styles from "./WhatsNewContent.module.scss";
 
 export type Props = {
   heading: string;
   description: string;
   image: string;
   videos?: Record<string, string>;
+  cta?: {
+    type?: "upgrade" | undefined;
```

Comment:
The `?` roughly means ` | undefined`, so the latter is redundant :)

```suggestion
    type?: "upgrade";
```

Reply:
That said, I feel like `<WhatsNewContent>` knows too much about the content of the CTA here; intuitively, I'd say the Call To Action could be anything, e.g. an upgrade link, but also just a link to an updated FAQ or something.

Wouldn't it be possible to move these conditions (`hasPremium`, `isPremiumAvailableInCountry`, etc.) into the `WhatsNewEntry` for the phone announcement, and have that use those conditions to determine what values it passes to the `cta` property?

```suggestion
```